app/controllers/transcriptions_controller.rb

```diff
@@ -67,11 +68,19 @@ def type_invalid?
     params[:data][:type] != "transcriptions"
   end
 
+  def whitelisted_attributes?
+    params[:data][:attributes] && params[:data][:attributes].keys.all? { |key| update_attr_whitelist.include? key }
```

Comment:
```suggestion
    update_attrs.keys.all? { |key| update_attr_whitelist.include? key }
```
Personally I think it'd be cleaner to check the attributes using the `update_attrs` method for the sake of readability, `update_attrs` gives you the data contained in `params[:data][:attributes]`.